doorman/application.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 import os
+import sys
```

Comment:
See comment below about generalizing the `StreamHandler` usecase.. Move this to heroku.cfg